README.md

```diff
@@ -6,15 +6,44 @@ A lightweight Merkle Drop contract.
 
 This project uses Hardhat.
 
-## Usage
+## Basic workflow
 
-### Deployment
+- Use the `deployMerkleDrop` task to deploy a MerkleDrop
+- Create a JSON balances file for each tranche
+- Use the `seedNewAllocations` task to add a new tranche with JSON balances
 
-    yarn task deploy --token 0xTOKEN --funders 0xFUNDER
+## Pro workflow
 
-### Adding a new tranche
+- Use `MerkleDropTranches` to track `MerkleDrop` contracts on the subgraph (task: `registerMerkleDrop`)
+- On frontends etc, use the subgraph and tranche URI to generate proofs for claims
 
-    yarn task add-tranche --contract 0xMERKLEDROP --json ./tranche1.json
+## Hardhat Tasks
+
+### Deploying MerkleDrop
+
+    yarn hardhat deployMerkleDrop --token 0x... --funders 0x...,0x...
+
+### Seeding new allocations (adding a tranche)
+
+    # From a local file:
+    yarn hardhat seedNewAllocations --merkle-drop 0x... --balances ./tranche0.json
+
+    # Or, fetch balances from a remote resource:
+    yarn hardhat seedNewAllocations --merkle-drop 0x... --balances https://raw.githubusercontent.com/mstable/stkBPT-merkle-drops/master/tranches/1/tranche-1.json
+
+### Deploying MerkleDropTranches
+
+    yarn hardhat deployMerkleDropTranches
+
+### Registering a MerkleDrop on MerkleDropTranches (useful for subgraphs)
+
+    yarn hardhat registerMerkleDrop --merkle-drop-tranches 0x... --merkle-drop 0x...
+
+### Adding a new tranche URI (useful for subgraphs)
+
+    yarn hardhat addTrancheURI --merkle-drop-tranches 0x... --merkle-drop 0x... --id 0  --balances ./tranche0.json
+
+## Development
```

Comment:
I would add here some of the steps to compile the project locally

```
yarn typechain
yarn compile
```

Testing could be part of `## Development` and `### Installation in your project` should a new section.

What do you think?

Reply:
I'll add it 👍